docs/Internal Systems/Gatsby_Starter_Guide.md

```diff
@@ -0,0 +1,28 @@
+# Gatsby Starter Guide
+Gastby is static site generator based on react. To boost performance, GatsbyJS loads only critical HTML, CSS, JavaScript, and other data. Once loaded, it prefetches resources for other pages you might navigate to. This way, your site loads as fast as possible and user interactions feel seamless. 
+
+The main thing about our project and Gatsby is that we are using no database and no server side code. Now, to use gatsby you need understanding of ReactJS and JavaScript. The videos down below gives a good overview of react. 
+
+
+https://www.youtube.com/watch?v=hQAHSlTtcmY&&t=897s
+
+
+[https://youtu.be/hQAHSlTtcmY?t=897s](https://youtu.be/hQAHSlTtcmY?t=897s)
```

Comment:
You could use HTML to include the video as an embedded frame?

Just a thought.
If not I would recommend labelling the link as the video title rather than the url